pygccxml/declarations/comment.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2014-2017 Insight Software Consortium.
+# Copyright 2004-2009 Roman Yakovenko.
+# Distributed under the Boost Software License, Version 1.0.
+# See http://www.boost.org/LICENSE_1_0.txt
+
+"""
+Describe a C++ comment declaration.
+
+"""
+
+
+from . import declaration
+
+
+class comment_t(declaration.declaration_t):
+
+    def __init__(self, name='', declarations=None):
+        """
+        Creates an object that describes a C++ comment declaration.
+
+        Args:
+
+        """
+        declaration.declaration_t.__init__(self, name)
+        self.location = {}
```

Comment:
Is there a reason this isn't piped through properties too?

Reply:
No, there isn't.  Fixed.